<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid red</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>173</width>
         <height>86</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_ip"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_port"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_startS">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>开启服务</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="test">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>197</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>发送命令</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbtnAnalysis">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>分析既有数据</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblCurIndex">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid red</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="test_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>刷新设备号</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCurDay">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当天数据</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbCurMonth">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>当月数据</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbOther">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>指定日期</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbtnSetDate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneDate">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>310</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QListWidget" name="lsDeviceType">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>390</y>
         <width>121</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="lsDepartment">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>111</width>
         <height>127</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>供电段</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>370</y>
         <width>72</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>传感器</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="expandPushButton" name="pbtnUIExpand">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid;
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>IP</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="tbIP">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>expandPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">expandpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
